torch/utils/data/datapipes/iter/readlinesfromfile.py

```diff
@@ -0,0 +1,18 @@
+from torch.utils.data import IterDataPipe
+
+
+class ReadLinesFromFileIterDataPipe(IterDataPipe[str]):
+    r""" :class:`ReadLinesFromFileDataPipe`
+
+    Iterable DataPipe to yield line(s) from the given file path.
+    args:
+        filepath : file path
+    """
+
+    def __init__(self, filepath):
+        self.filepath = filepath
```

Comment:
Can we take a prior datapipe like LoadFilesFromDisk or ListDirFiles as input rather than one file path?

Reply:
fyi yes, it now takes an iterdatapipe<string>